variables.tf

```
@@ -66,6 +66,18 @@ variable "resource" {
   description = "Hive resources"
 }
 
+variable "resource_proxy" {
+  type = object({
+    cpu     = number,
+    memory  = number
+  })
+  default = {
+    cpu     = 500,
+    memory  = 1024
+  }
```

Could we possibly have this a little lower as default? Thinking that people might run this on their local machine as well. 
```suggestion
  default = {
    cpu     = 300,
    memory  = 512
  }
```

---

Agree.